src/chocolatey/GetChocolatey.cs

```diff
@@ -49,8 +49,9 @@ public class GetChocolatey
 
         public GetChocolatey()
         {
+            Log4NetAppenderConfiguration.configure();

```

Comment:
It gets set in the container registration: https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/registration/ContainerBinding.cs#L45

Let me know if you want to change this. Seemed like this was a sensible default. It threw me off initially having `run` not emit anything.